src/components/Strawpoll.tsx

```diff
@@ -121,6 +122,17 @@ export default function Strawpoll(props: StrawpollProps) {
       strawpollFref.child('options').push(DEFAULT_STRAWPOLL_OPTION);
     }
 
+    const togglePollType = () => {
+      strawpollFref.child('type').set(type === StrawpollType.Checkbox ? StrawpollType.Radio : StrawpollType.Checkbox);
+      // reset votes
```

Comment:
if we're currently in `type === StrawpollType.Radio` transitioning to `Checkbox`, there's an argument to be made that you don't necessarily have to reset the votes, just turn them into checkbox votes.

Obviously that means that votes get reset in only one situation here, which might be confusing. Wondering what your thoughts are on this.

Reply:
That's true, from a validity point it's not required, but I do think being consistent is probably more important. Do you know if it's common to go back and edit settings after sharing the poll?

Reply:
It's something that could happen to be honest - I'm imaging a scenario where a director makes a radio poll and then someone gets upset that they couldn't select more options, and a back and forth between the proposer and the complainer and the director ensue.